cypress/e2e/functional/researcher_permission_forms.spec.js

```diff
@@ -0,0 +1,113 @@
+        cy.login(RESEARCHER_USERNAME, c.RESEARCHER_PASSWORD); // Login as researcher user
+    });
+    
+    after(function() {
+        cy.clearAllCookies();
+        cy.ge
+    });
+
+    it("Tests Permission forms", () => {
+        cy.log('Verify, and then click, the Admin link');
+        pf.verifyAdminLink();
+        pf.clickAdminLink();
+
+        cy.log('Verify, and then click, the Permission Forms link');
+        pf.verifyPermissionFormsLink();
+        pf.clickPermissionFormsLink();
+
+        cy.log('Verify that both links exist; but only the Create / Manage Project Permission Forms link is enabled');
+        pf.verifyManageStudentPermissionsLink();
+        pf.verifyManageStudentPermissionsLinkEnabled(false);
+        pf.verifyCreateManageProjectPermissionFormsLink();
+        pf.verifyCreateManageProjectPermissionFormsLinkEnabled();
+
+        cy.log('Click the Create / Manage Project Permission Forms link; then verify that only the Manage Student Permissions link is enabled');
+        pf.clickCreateManageProjectPermissionFormsLink();
+        pf.verifyManageStudentPermissionsLinkEnabled();
+        pf.verifyCreateManageProjectPermissionFormsLinkEnabled(false);
+
+        cy.log('Click the Create New Permission Form button, fill in values in the dialog; but then click Cancel');
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME, 'google.com', true);
+
+        cy.log('Fill in the Create New Permission Form dialog again, but this time, click the Save button');
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL,
+            false, TEST_PROJECT_NAME);
+
+        cy.log('Create more Permission Forms, to be used to test the Archive, Unarchive & Delete buttons');
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL,
+            false, TEST_PROJECT_NAME);
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL,
+            false, TEST_PROJECT_NAME);
+        pf.createNewPermissionForm(TEST_PROJECT_NAME, TEST_PERMISSION_FORM_NAME+'4', 'learn.portal.staging.'+TEST_PERMISSION_FORM_URL,
+            false, TEST_PROJECT_NAME);
+            
+        cy.log("Select a Project from the dropdown (on the 'Create New Permission Form' tab)");
+        pf.selectCreateManagePermissionsProject(TEST_PROJECT_NAME);
+
+        cy.log('Test the links on the Permission Forms table, starting with URL text, Archive, and Unarchive');
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'1', TEST_PERMISSION_FORM_URL);
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'2', 'www.'+TEST_PERMISSION_FORM_URL);
+        pf.verifyPermissionFormUrlText(TEST_PERMISSION_FORM_NAME+'3', 'learn.'+TEST_PERMISSION_FORM_URL);
+
+        // TODO: fix clicking on archived rows, then uncomment these:
```

Comment:
@johnTcrawford - if this is a known bug, please be sure to reference the PT number in the comments.

Reply:
Sorry that this was unclear: I have to fix my automation code so that it can click on rows that have been archived; there is no Portals bug here, it works fine manually.